server/scsynth/SC_GraphDef.cpp

```diff
@@ -246,6 +246,60 @@ void GraphDef_ReadVariant(World *inWorld, char*& buffer, GraphDef* inGraphDef, G
 	}
 }
 
+
+typedef struct IndexMap {
+	uint32 index;
+	uint32 mIndex;
```

Comment:
what is index, what is mIndex? m is a common prefix for "member", though both are members. maybe there are more expressive names?